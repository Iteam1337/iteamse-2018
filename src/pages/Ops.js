// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/OperationsPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import Header from '../components/Header/Header'
import opsImage from './img/illustrations-group-4.svg'
import GridColumn from '../components/Grid/GridColumn'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import Team from '../components/Team/Team'

type Props = Iteam.ApolloBase<IteamCMS.OperationsPage>

export const OperationsPageQuery = gql`
  query OperationsPage {
    pageOps {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      aboutText
      aboutTitle
      contentImage
      hardwareText
      hardwareTitle
      networkText
      networkTitle
      softwareText
      softwareTitle
    }
  }
`

const Operations = () => {
  return (
    <Query query={OperationsPageQuery}>
      {({ loading, data: { pageOps } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageOps.headerImage}
              messageBgColor={pageOps.headerTextBgColor}
              messageOne={pageOps.headerText1}
              messageTwo={pageOps.headerText2}
            />

            <GridColumn>
              <Block title={pageOps.aboutTitle}>{pageOps.aboutText}</Block>

              <ImageBleed image={pageOps.contentImage} />

              <Block concrete title={pageOps.networkTitle}>
                {pageOps.networkText}
              </Block>

              <ImageBlock image={opsImage} />

              <Block concrete title={pageOps.softwareTitle}>
                {pageOps.softwareText}
              </Block>

              <Block concrete title={pageOps.hardwareTitle}>
                {pageOps.hardwareText}
              </Block>
            </GridColumn>

            <Team bgColor="red" shortName={['rln', 'msr']} />
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Operations
