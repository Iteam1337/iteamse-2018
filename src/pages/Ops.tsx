import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { OperationsPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import { set } from '../utils/googleAnalytics'
import opsImage from './img/illustrations-group-4.svg'

export const OPERATIONS_PAGE_QUERY = gql`
  query OperationsPage {
    pageOps {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      aboutText
      aboutTitle
      contentImage
      hardwareText
      hardwareTitle
      networkText
      networkTitle
      softwareText
      softwareTitle
      team
      contactTitle
    }
  }
`

class OperationsQuery extends Query<OperationsPageQuery> {}

export class Operations extends React.Component {
  componentDidMount() {
    set('ops')
  }

  render() {
    return (
      <OperationsQuery query={OPERATIONS_PAGE_QUERY}>
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { pageOps } = data

          return (
            <>
              <Helmet>
                <title>Iteam |Â Operations</title>
                <meta name="og:title" content="Iteam | Operations" />
                <meta name="twitter:title" content="Iteam | Operations" />
                {pageOps.headerImage && (
                  <meta
                    name="og:image"
                    content={`https:${pageOps.headerImage}`}
                  />
                )}
              </Helmet>
              <Header
                backgroundImage={pageOps.headerImage}
                messageBgColor={pageOps.headerTextBgColor}
                messageOne={pageOps.headerText1}
                messageTwo={pageOps.headerText2}
              />

              <GridColumn>
                <Block title={pageOps.aboutTitle}>{pageOps.aboutText}</Block>

                <ImageBleed image={pageOps.contentImage} />

                <Block title={pageOps.networkTitle}>
                  {pageOps.networkText}
                </Block>

                <ImageBlock image={opsImage} />

                <Block title={pageOps.softwareTitle}>
                  {pageOps.softwareText}
                </Block>

                <Block title={pageOps.hardwareTitle}>
                  {pageOps.hardwareText}
                </Block>
              </GridColumn>

              <Team
                bgColor="red"
                shortName={pageOps.team}
                callToAction={pageOps.contactTitle}
              />
            </>
          )
        }}
      </OperationsQuery>
    )
  }
}

export default Operations
