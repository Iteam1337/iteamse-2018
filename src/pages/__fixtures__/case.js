export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
