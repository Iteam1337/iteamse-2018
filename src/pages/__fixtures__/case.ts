export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
