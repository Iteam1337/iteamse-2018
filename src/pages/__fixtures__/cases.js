export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
