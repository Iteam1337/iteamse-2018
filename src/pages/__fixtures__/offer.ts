// tslint:disable:object-literal-sort-keys
export const offers = [
  {
    offerIntroImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5n7J4mzx3UWam0YwgsUQm/8e00d12d6e80eba927b2a313fb250d87/1.png',
    offerTitle: 'Kod',
    offerIntroText:
      'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med. ',
    offerLead:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. ',
    offerUspOneTitle: 'Utveckling av pilot',
    offerUspOneText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspTwoTitle: 'API',
    offerUspTwoText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspThreeTitle: 'Mobila applikationer',
    offerUspThreeText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspFourTitle: 'AI-lösningar',
    offerUspFourText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi\n',
    offerIllustrationImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5SAhc8YuL6OaQQsC6miokK/7c09d594e44ffa47781bf1bdb21aae7f/illustrations-group-2.png',
    offerOrder: 1,
    __typename: 'Offer',
  },
]
