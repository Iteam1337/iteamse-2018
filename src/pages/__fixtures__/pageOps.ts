export const pageOps = {
  __typename: 'PageOps',
  aboutText:
    'Operations är namnet på Iteams drift & supportavdelning, sedan 1995 har vi hjälpt våra kunder att göra det de ska göra - utan att vara hindrade av krånglande teknik. Vårt mål är att erbjuda en personlig, professionell service - utan någon “First line” att ta sig förbi.\n\nVi är inte bundna till någon specifik leverantör av hård- eller mjukvara utan rekommenderar alltid det som är bäst i varje enskilt fall. Beroende på uppdraget har Iteam varit allt från komplett uthyrd IT avdelning till specialisthjälp inom olika områden.\n',
  aboutTitle: 'Om oss',
  contactTitle: 'Hör av dig till oss!',
  contentImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2CDS7p7sxGw0GyqSwqyQO6/64a78af7e16d70db815cd8677244c5a9/bleed_ops.jpg',
  hardwareText:
    'Iteam har lång erfarenhet av att installera och underhålla redundanta system för virtualisering. Vi är mest hemma i Microsoft Hyper-V kluster men klarar även andra.\nVi hjälper dig välja utrustning för just dina behov, vi är märkesoberoende även om vi har våra favoriter. Vi har erfarenhet av arbetsstationer och servrar från bl.a Dell, Lenovo, HP, Supermicro och är certifierade på QSAN för lagringslösningar.\n',
  hardwareTitle: 'Hårdvara',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/29FbYXDqLu2CAkCKEIAuMe/b33442d0204941ae1acda8b7134206d5/ops.jpg',
  headerText1: 'Skräddarsydd drift och support',
  headerText2: 'sedan 1995',
  headerTextBgColor: '#FF405F',
  networkText:
    'Oavsett om det gäller High Availability med site-to-site VPN eller trådlösa nätverk med 802.1X så kan vi hjälpa dig och ditt företag. Vi är partner och certifierade på svenska leverantören Clavisters brandväggsprodukter. \n\nVi använder i huvudsak Ubiquiti Networks nätverksprodukter och är certifierade på produktserien UniFi men har även erfarenhet produkter från Dell, HP/Aruba och Meraki.\n',
  networkTitle: 'Nätverk',
  softwareText:
    'Som Microsoft Silver Partner hjälper vi er med hela er Windows-miljö, servrar och klienter. Vi automatiserar installation och uppdateringar av mjukvaran ni använder i ert dagliga arbete. Vi har med största sannolikhet erfarenhet av den mjukvara ni använder, ja - även den där som bara funkar i Windows NT på en gammal dator i skamvrån.\n\nPS. Vi är även Red Hat Partner och hjälper er med era Linux-servrar.\n',
  softwareTitle: 'Mjukvara',
  team: ['rln', 'msr'],
}
