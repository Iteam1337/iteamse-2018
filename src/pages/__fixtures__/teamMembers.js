export const teamMembers = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'Stockholm',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    short: 'cmm',
    __typename: 'TeamMember',
  },
  {
    avatar: '',
    email: 'count@sesame.com',
    gravatar: 'count.png',
    location: 'GÃ¶teborg',
    name: 'Count van Count',
    phoneNumber: '0707654321',
    title: '1 ahahah',
    short: 'cvc',
    __typename: 'TeamMember',
  },
]
