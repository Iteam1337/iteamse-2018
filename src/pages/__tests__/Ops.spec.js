import React from 'react'
import Ops from '../Ops'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageOps } from '../__fixtures__/pageOps'
import { teamMembers } from '../__fixtures__/teamMembers'

describe('components/Ops', () => {
  const mockedResponse = {
    pageOps,
    teamMembers,
  }

  it('renders Ops', async () => {
    const { container } = render(
      <MockedQuery response={mockedResponse}>
        <Ops />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
