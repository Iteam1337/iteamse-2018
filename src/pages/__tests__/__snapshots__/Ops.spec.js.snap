// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Ops renders Ops 1`] = `
<div>
  <div
    style="height: 700px;"
    class="lazyload-placeholder"
  />
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD cdtVAw"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Om oss
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Operations är namnet på Iteams drift & supportavdelning, sedan 1995 har vi hjälpt våra kunder att göra det de ska göra - utan att vara hindrade av krånglande teknik. Vårt mål är att erbjuda en personlig, professionell service - utan någon “First line” att ta sig förbi.
                </p>
                <p>
                  Vi är inte bundna till någon specifik leverantör av hård- eller mjukvara utan rekommenderar alltid det som är bäst i varje enskilt fall. Beroende på uppdraget har Iteam varit allt från komplett uthyrd IT avdelning till specialisthjälp inom olika områden.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <img
      class="ImageBleed__ImageContainer-vnhpmd-0 fRWLkh"
      src="//images.ctfassets.net/rj4r6yfcesw5/2CDS7p7sxGw0GyqSwqyQO6/64a78af7e16d70db815cd8677244c5a9/bleed_ops.jpg"
    />
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Nätverk
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Oavsett om det gäller High Availability med site-to-site VPN eller trådlösa nätverk med 802.1X så kan vi hjälpa dig och ditt företag. Vi är partner och certifierade på svenska leverantören Clavisters brandväggsprodukter. 
                </p>
                <p>
                  Vi använder i huvudsak Ubiquiti Networks nätverksprodukter och är certifierade på produktserien UniFi men har även erfarenhet produkter från Dell, HP/Aruba och Meraki.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Mjukvara
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Som Microsoft Silver Partner hjälper vi er med hela er Windows-miljö, servrar och klienter. Vi automatiserar installation och uppdateringar av mjukvaran ni använder i ert dagliga arbete. Vi har med största sannolikhet erfarenhet av den mjukvara ni använder, ja - även den där som bara funkar i Windows NT på en gammal dator i skamvrån.
                </p>
                <p>
                  PS. Vi är även Red Hat Partner och hjälper er med era Linux-servrar.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="PaddedRow-s3nsc5b-0 cKRLCb"
        data-test="block-"
      >
        <div
          class="Block__Content-jdahsb-0 eJFCrU"
        >
          <div
            class="Block__TitleWrap-jdahsb-1 dkYDLt"
          >
            <h3
              class="H3-s19jy0q4-0 bFoQlU"
            >
              Hårdvara
            </h3>
          </div>
          <div
            class="Block__Text-jdahsb-3 bdZkSQ"
          >
            <div
              class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
            >
              <div>
                <p>
                  Iteam har lång erfarenhet av att installera och underhålla redundanta system för virtualisering. Vi är mest hemma i Microsoft Hyper-V kluster men klarar även andra. Vi hjälper dig välja utrustning för just dina behov, vi är märkesoberoende även om vi har våra favoriter. Vi har erfarenhet av arbetsstationer och servrar från bl.a Dell, Lenovo, HP, Supermicro och är certifierade på QSAN för lagringslösningar.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD dXaaGv"
  >
    <div
      class="TeamMembers__Members-gavpkf-0 kEUlpQ"
    >
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 foLTeb"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cmm"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              Cookie master
            </div>
            <div>
              Stockholm
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Cookie M
          </div>
          <div>
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="tel:0701234567"
            >
              070-123 45 67
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:cm@sesame.com"
          >
            cm@sesame.com
          </a>
        </div>
      </div>
      <div
        class="TeamMemberDuo__Colleague-s9kjazo-0 foLTeb"
      >
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cvc"
        >
          <div
            style="height: 226px;"
            class="lazyload-placeholder"
          />
        </a>
        <div
          class="TeamMemberDuo__Meta-s9kjazo-1 gzvJxw"
        >
          <div
            class="TeamMemberDuo__MetaSection-s9kjazo-3 XeOZD"
          >
            <div
              class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
            >
              1 ahahah
            </div>
            <div>
              Göteborg
            </div>
          </div>
          <div
            class="TeamMemberDuo__Title-s9kjazo-2 dFgPDl"
          >
            Count van Count
          </div>
          <div>
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="tel:0707654321"
            >
              070-765 43 21
            </a>
          </div>
          <a
            class="NativeLink-s1qo6rwq-0 jNinQt"
            href="mailto:count@sesame.com"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
